src/css/tabs/osd.less

```diff
@@ -349,8 +349,8 @@
 		}
 		.osd-preview {
 			flex: 2 0 auto;
-			width: 350px;
-			max-width: max(min(950px,calc((100vh - 470px) * 1.25)), 350px);
+			width: 740px;
+			max-width: max(min(950px,calc((100vh - 470px) * 1.25)), 740px);
```

Comment:
The problem here is the "size" of the char that contains the symbol. It is 12x18 and we don't reduce it because it almost disappears because the reduction algorithm done by the browser works very bad with character images.
Until now, PAL and NTSC have 30 columns, so 30 x 12 = 360 px (the CSS rule was wrong by 10 pixels). Now, in HD it contains 53 columns, so the minimum must be 53 x 12 = 636.
This will make difficult to fit all without flexing, so the better approach is to give different sizes for both systems, at least PAL and NTSC will fit better. We have the code to add a class, and use different backgronds depending on the video system before this PR https://github.com/betaflight/betaflight-configurator/pull/3071 that removed it because it was not needed anymore.
So, two options:
- Solution 1: simply use 636 as maximum
- Solution 2: add the code removed in the PR to add a class based on the video system, and use 360 by default and 636 for HD.

Reply:
Thanks. Using solution 1 now. Going to check solution 2.